build.gradle

```diff
@@ -201,6 +203,33 @@ def allSubProjectsConfig = {
                         details.because "'org.springframework.extensions.surf' was moved to 'org.alfresco.surf'"
                     }
                 }
+
+                // Eclipse Gemini Blueprint 1.0.2.RELEASE was published as 'org.eclipse.gemini' and is
+                // the last version that supports Spring 3.x
+                // Starting from 2.0 the modules were moved to 'org.eclipse.gemini.blueprint' and artifacts
+                // were renamed.
+                if (details.requested.group == 'org.eclipse.gemini.blueprint'
+                    && details.requested.version.startsWith('1.0')) {
```

Comment:
@vierbergenlars Is this better ?

Reply:
Yes, I do think this looks better and more easy to understand the idea.